### ISSUE TITLE
Class comments are dropped by Spoon when reading from file system

### ISSUE THREAD

**Issue by slarse** [KTH/spork#18]
I'm not sure if this is something I'm doing wrong, or if Spoon is doing it wrong (I think the latter), but adding an input resource from the file system results in the class comment being dropped. Not always, but often. For example, with this file:

```java
/**
 * This is a comment!
 */
public class Test {}
```

Then this drops the class comment:
```java
launcher.addInputResource(new FileSystemFile(pathToFile));
```
but this captures the class comment:
```java
launcher.addInputResource(new VirtualFile(readFile(pathToFile)));
```

TODO: Work around for now, report on Spoon issue tracker if the latest build doesn't resolve it.

**Comment by slarse** [comment]
This is related to inria/spoon#3300

**Comment by slarse** [comment]
The problem is that the comments are attached to the compilation unit if they are at the very start of the file. A workaround could be to simply attach them to the package statement instead (if there is a package statement), and simply not care about that odd case where there is no package statement.

**Comment by slarse** [comment]
Fixed by 3ceee85ff918494308b9bc2a5ffd9cd323b02bfb
